leap_c/task.py

```diff
@@ -0,0 +1,44 @@
+from abc import ABC, abstractmethod
+from typing import Any, Optional
+
+import gymnasium as gym
+import torch
+
+from leap_c.mpc import MPC, MPCBatchedState, MPCInput
+
+
+class Task(ABC):
+    """A task is a wrapper around an gym environment and an MPC planner."""
+    def __init__(self, mpc: MPC, env: gym.Env):
+        """A task is a wrapper around an gym environment and an MPC planner.
```

Comment:
Should this really be the same as the class description? I forgot what exactly our policy for docstring in class vs docstring in __init__ was :)